src/main/java/ua/vedroid/cinema/security/impl/AuthenticationServiceImpl.java

```diff
@@ -1,27 +1,34 @@
 package ua.vedroid.cinema.security.impl;
 
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import ua.vedroid.cinema.model.User;
 import ua.vedroid.cinema.security.AuthenticationService;
+import ua.vedroid.cinema.service.RoleService;
 import ua.vedroid.cinema.service.ShoppingCartService;
 import ua.vedroid.cinema.service.UserService;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
     private final UserService userService;
     private final ShoppingCartService shoppingCartService;
+    private final RoleService roleService;
 
     @Autowired
     public AuthenticationServiceImpl(UserService userService,
-                                     ShoppingCartService shoppingCartService) {
+                                     ShoppingCartService shoppingCartService,
+                                     RoleService roleService) {
         this.userService = userService;
         this.shoppingCartService = shoppingCartService;
+        this.roleService = roleService;
     }
 
     @Override
     public User register(String email, String password) {
-        User user = userService.add(new User(email, password));
+        User newUser = new User(email, password);
```

Comment:
Everything is OK but I can't find on what level are you encrypting passwords. Are they encrypted?

Reply:
Yes, they are encrypted. In `UserServiceImpl.add()`